db/migrate/20201125151816_create_schools.rb

```diff
@@ -0,0 +1,18 @@
+class CreateSchools < ActiveRecord::Migration[6.0]
+  def change
+    create_table :schools, id: :uuid do |t|
+      t.timestamps
+      t.column :urn, :string, null: false
+      t.column :name, :string, null: false
+      t.column :address, :string, null: false
```

Comment:
Not sure about this. I haven't seen many applications where storing the address as just one string is useful. Perhaps we need to discuss what we might use the address.

Reply:
I am pretty sure most of our data model will need overhauling later down the line (I had a chat about that this morning with Chris, he and Brandon are working on a more fully-fledged entity model). We aren't using address for much atm, it's just there as a placeholder. Kind of like number of pupils, school type, and boolean flags.

Reply:
@kicferk1 of course most of the data model will change though probably having a separate address table with t.string "address_line1"
    t.string "address_line2"
    t.string "address_line3"
    t.string "address_line4"
    t.string "country"
    t.string "postcode" is probably a better starting point. Happy to approve. I'll leave the final call to you.